Tests/Block/PageListBlockServiceTest.php

```diff
@@ -29,7 +29,7 @@ protected function setUp()
     {
         parent::setUp();
 
-        $this->pageManager = $this->getMock('Sonata\PageBundle\Model\PageManagerInterface');
+        $this->pageManager = $this->createMock('Sonata\PageBundle\Model\PageManagerInterface');
```

Comment:
does AbstractBlockServiceTestCase extend the new TestCase? I mean, related to the failing travis build (--prefer-lowest)?